lib/taskana-core/src/main/java/pro/taskana/jobs/TaskCleanupJob.java

```diff
@@ -107,10 +110,12 @@ public void run() throws TaskanaException {
                 .list();
         }
 
+        LOGGER.debug("exit from getTasksCompletedBefore(), returning {}", taskList);
         return taskList;
     }
 
     private int deleteTasksTransactionally(List<TaskSummary> tasksToBeDeleted) {
+        LOGGER.debug("entry to deleteTasksTransactionally(tasksToBeDeleted = {})", LoggerUtils.listToString(tasksToBeDeleted));
```

Comment:
please invoke this only conditionally (if LOGGER.isDebugEnabled() ), because the tasksToBeDeleted list might be large and therefore, the listToString operation should only performed if it is really needed

Reply:
Done

Reply:
Done